fuzzing/tools/launcher.py

```diff
@@ -11,23 +26,31 @@
 FLAGS = flags.FLAGS
 
 flags.DEFINE_integer(
-    'timeout_secs',
+    "timeout_secs",
     20,
-    'The maximum duration, in seconds, of the fuzzer run launched.',
+    "The maximum duration, in seconds, of the fuzzer run launched.",
     lower_bound=0)
 
+flags.DEFINE_string("corpus_dir", "", "The directory containing corpus.")
```

Comment:
We can make the documentation of the flag more precise to suggest it is optional: `If non-empty, a directory that will be used as a seed corpus for the fuzzer run.`